system/security/plugins/htpasswdSecurity/htpasswdSecurity.cpp

```diff
@@ -25,11 +25,13 @@
 class CHtpasswdSecurityManager : public CBaseSecurityManager
 {
 public:
-    CHtpasswdSecurityManager(const char *serviceName, IPropertyTree *authconfig) : CBaseSecurityManager(serviceName, (IPropertyTree *)NULL)
+    CHtpasswdSecurityManager(const char *serviceName, IPropertyTree *secMgrCfg, IPropertyTree *authconfig) : CBaseSecurityManager(serviceName, (IPropertyTree *)NULL)
 	{
-		if (authconfig)
-			authconfig->getProp("@htpasswdFile", pwFile);
-		apr_initialized = false;
+        if (secMgrCfg)
```

Comment:
Its a tab thing, the legacy was tabbed and the new stuff wasn't so it looked horrible. I didn't want to untabify the whole thing so I just did the body of the constructor